### PR TITLE
Fixes drilldown per item day and descending order on agg by item

### LINKED ISSUE #2276: Cloud Cost item details show the daily totals for all items
Drilling down into "Items" in the Breakdown provides a list of all Items available, but clicking on an individual item has a details popup that is labeled with the individual item but has the daily totals for all items.

Screenshot 2023-11-01 at 11 24 39 PM

### LINKED ISSUE #2278: Cloud Cost "Item" breakdown results should default to descending sort order
Items [currently return all results](https://github.com/opencost/opencost/issues/2274) and aren't sorted at all. Everything else appears to default to descending sort order.

### PR DESCRIPTION
**Pull request by jjarrett21**
## What does this PR change?
* Fixes filtering so data is right on drilldown and agg by item is in descending order

## Does this PR relate to any other PRs?
* no

## How will this PR impact users?
* they'll have more accurate drilldown and be able to easily view cost in the table at it's largest to smallest. 

## Does this PR address any GitHub or Zendesk issues?
* Closes #2276
* Closes #2278 

## How was this PR tested?
* self tested

## Does this PR require changes to documentation?
* no

## Have you labeled this PR and its corresponding Issue as "next release" if it should be part of the next OpenCost release? If not, why not?
* yes 
Screenshot 2023-11-03 at 10 41 52 PM
Screenshot 2023-11-03 at 10 41 46 PM

### CODE REVIEW

File: .github/workflows/label-comments.yml
Comment (mattray) [inline .github/workflows/label-comments.yml:1]: Why was this part of the PR? It seems unrelated

Comment (jjarrett21) [reply .github/workflows/label-comments.yml:1]: It was just there when I pulled develop down

**Review by michaelmdresser — APPROVED** [review]

**Comment by mattray** [comment]
Running this against the previous `cloudcost` image fixes the sorting but the item drilldown gives a 400 error. In the logs it looks like the queries are hitting URLs like

```
127.0.0.1 - - [30/Nov/2023:06:01:24 +0000] "GET /model/cloudCost?window=7d&costMetric=AmortizedNetCost&filter=providerIds:%22gke-alex-skunkworks-2-pool-1-80d6ba8b-2cpx%22 HTTP/1.1" 400 105 "http://localhost:9090/cloud?agg=item" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.1 Safari/605.1.15" "-"
```
note that working queries look like
```
127.0.0.1 - - [30/Nov/2023:06:01:04 +0000] "GET /model/cloudCost/view/graph?window=7d&aggregate=service&costMetric=AmortizedNetCost&filter=provider:%22Azure%22&limit=1000 HTTP/1.1" 200 1836 "http://localhost:9090/cloud" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.1 Safari/605.1.15" "-"
```
where the `/model/cloudCost/view/` works but `/model/cloudCost?` doesn't. I'm not familiar enough with the code to fix it right now.

**Review by mattray — CHANGES_REQUESTED** [review]
Let's verify this works before merging.

**Comment by mattray** [comment]
OK, there appears to be a race condition between the data being pulled down from the cloud bills and the UI and eventually gets populated with data. The good news is this patch fixes clicking on the Provider (ie. GCP) and getting filtered Services (by GCP) and clicking on a Service and getting filtered Items for that Service. The bad news is the Item details show that Item in the pop-up but the graph is for the entire Service. The other issue is that if you select a new Breakdown option (ie. Provider) it maintains the previous filter (only showing GCP).

**Comment by mattray** [comment]
This was incorporated into https://github.com/opencost/opencost/pull/2375

**Pull request closed without merging**